ecs_deploy/ecs.py

```diff
@@ -214,7 +214,7 @@ def parse_command(command):
                     "command: {} resulting in error: {}"
                     .format(command, str(e)))
 
-        return command.split(' ')
+        return [cmd for cmd in command.split() if cmd]
```

Comment:
Hi @normoes, what I meant was this – it's the same result as your line:
```suggestion
        return command.split()
```

Reply:
Now I get it ;)

Reply:
Thanks! 🎉